src/utils/file-system.ts

```diff
@@ -0,0 +1,68 @@
+import { promises as fs } from "fs";
+
+/**
+ * @description Function used to read a file from the file system
+ * @param path - The absolute path to the file
+ * @returns A buffer of the file or `undefined` if no file found
+ */
+export async function readFile(path: string): Promise<Buffer | void> {
+  try {
+    return await fs.readFile(path);
+  } catch (error: unknown) {
+    if (error instanceof Error) {
+      console.error(`Got an error trying to read the file: ${error.message}`);
+    } else {
+      console.error(error);
+    }
+  }
+}
+
+/**
+ * @description Function used to write a new file to the file system
+ * @param path - The absolute path to the file
+ * @param data - A buffer containing your file data
+ * @returns A buffer of the file or `undefined` if no file found
+ */
+export async function writeFile(path: string, data: Buffer): Promise<Buffer | void> {
+  try {
+    return await fs.writeFile(path, data);
+  } catch (error: unknown) {
+    if (error instanceof Error) {
+      console.error(`Got an error trying to read the file: ${error.message}`);
+    } else {
+      console.error(error);
+    }
+  }
+}
+
```

Comment:
What's the reason behind this file, why would we need to read, write into a file?

Reply:
To save the cover `img` and `pdf` on the file system